src/backend/api/meals.js

```diff
@@ -56,7 +56,9 @@ router.put("/:id", async(request, response)=> {
 router.post("/", async (request, response) => {
   const newMeal = request.body;
   const insertedMeal = await knex("Meal").insert({
```

Comment:
I think normally this should be lowercase for databases, but not sure it is worth fixing right now